wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearningReview/ActiveLearningReviewContainer.vue

```diff
@@ -152,8 +158,14 @@ export default Vue.extend({
         },
         booleanOperator() {
             this.showFlags = !!this.firstComparison && !!this.booleanOperator;
+        },
+        userSelections() {
+            this.updateFilteredSortedGroupedSuperpixels();
         }
     },
+    mounted() {
+        this.updateFilteredSortedGroupedSuperpixels();
```

Comment:
Doesn't this need to be called when activated (so it updates when something has changed during guided mode)?

Reply:
I suppose rather than removing the deep watcher, you could skip the update if the component is deactivated and set a flag that it needs an update when reactivated (and then use that conditional on activation).

Reply:
I originally tried that approach. Unfortunately there's no way to stop the deep comparison as long as the watcher exists, so this still results in a very obvious slowdown in guided mode once the review mode is loaded - even without recomputing the filtered list.